content/08-mongodb/06-managing-documents.mdx

```diff
@@ -475,3 +475,81 @@ Learning how to create, query for, update, and remove documents gives you the sk
 If you are using [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) with MongoDB, you can use the [MongoDB connector (currently in preview)](https://www.prisma.io/docs/concepts/database-connectors/mongodb) to connect and manage your data.
 
 </PrismaOutlinks>
+
+## FAQ
+
+<details><summary>What is an embedded document in MongoDB?</summary>
+
+An [embedded, or nested, document](https://www.mongodb.com/basics/embedded-mongodb) in MongoDB is a document which contains another document inside of it. 
+
+The following is an example of an embedded document where `address` is a subdocument that can be accessed with the `user` record.
+
+```
+db.user.findOne({_id: 111111})
+    
+{    
+    _id: 111111,    
+    email: “email@example.com”,    
+    name: {given: “Jane”, family: “Han”},    
+    address: {    
+    street: “111 Elm Street”,    
```

Comment:
Just wondering if it would be helpful to show an extra level of indentation for this nesting.  I guess the client doesn't return it with extra nesting, which is kind of annoying.  So I guess it's just a question of whether we want to show the output as-is or alter it for readability.  I wanted to bring it up, but leave the decision up to you as I think either would be reasonable choices.

Reply:
That's a good point.

Reply:
What do you think of adding language stating "...where `address`—denoted as a subdocument by additional curly brackets— can be accessed..."